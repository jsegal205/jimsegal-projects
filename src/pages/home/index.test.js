import React from "react";
import { BrowserRouter as Router } from "react-router-dom";
import { render, cleanup } from "@testing-library/react";
import Home from "./index";

afterEach(cleanup);

describe("Home Component", () => {
  it("renders correctly", () => {
    const { container } = render(
      <Router>
        <Home />
      </Router>
    );
    expect(container).toMatchSnapshot();
  });
});
