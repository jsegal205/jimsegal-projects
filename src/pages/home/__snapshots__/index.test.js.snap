// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home Component renders correctly 1`] = `
<div>
  <section>
    <ul
      class="project-list"
    >
      <li
        class="project-list-item"
        style="border-color: #cf2065;"
      >
        <a
          class="project-list-item-link"
          href="/congress"
        >
          <div
            class="project-list-item-placeholder"
            style="background-color: rgb(48, 223, 154);"
          >
            <h2
              class="project-list-item-title"
              style="color: rgb(68, 68, 68);"
            >
              Congress
            </h2>
          </div>
          <p>
            Some fun data surrounding the current congressional sesssion
          </p>
        </a>
      </li>
      <li
        class="project-list-item"
        style="border-color: #1101be;"
      >
        <a
          class="project-list-item-link"
          href="/recipes"
        >
          <div
            class="project-list-item-placeholder"
            style="background-color: rgb(238, 254, 65);"
          >
            <h2
              class="project-list-item-title"
              style="color: rgb(68, 68, 68);"
            >
              Recipes
            </h2>
          </div>
          <p>
            A bunch of recipes that I frequent and wanted to show off
          </p>
        </a>
      </li>
      <li
        class="project-list-item"
        style="border-color: #5c44d6;"
      >
        <a
          class="project-list-item-link"
          href="/spacex"
        >
          <div
            class="project-list-item-placeholder"
            style="background-color: rgb(163, 187, 41);"
          >
            <h2
              class="project-list-item-title"
              style="color: rgb(68, 68, 68);"
            >
              Space X
            </h2>
          </div>
          <p>
            A countdown timer until the next launch of a Space X rocket
          </p>
        </a>
      </li>
    </ul>
  </section>
</div>
`;
